votes/0639-propose-a-release-note-guide.md

```diff
@@ -0,0 +1,76 @@
+
+Depending on what your PR changes, you can refer to one of the following release note types:
+
+- Compatibility change
+- Improvement or Feature enhancement
+- Bug fix
+
+## Compatibility change
+
+A compatibility change note means:
+
+- Your PR adds, removes, or modifies one or more configuration items or system variables.
+- Your PR modifies the default value of a configuration item or system variable.
+
+For this type of note, you should clearly and adequately state the following aspects:
+
+- The previous code behavior, configuration item, or default value.
+- The new code behavior, configuration item, or default value since the new version.
+
+Note that the object of the change should be user-perceivable. If the changed configuration item or system variable is not supposed to be exposed to users, do not include it in your release notes.
+
+**Examples:**
+
+| **Not recommended**                                          | **Clear in type** | **Adequate and clear in meaning** | **User perspective** | Recommended                                                  |
+| ------------------------------------------------------------ | ----------------- | --------------------------------- | -------------------- | ------------------------------------------------------------ |
+| copr: cast invalid utf8 string to real bug fix               | ❌                 | ❌                                 | ❌                    | Previously, when TiDB converts an illegal UTF-8 string to a Real type, an error is reported directly. From now on, TiDB will process the conversion according to the legal UTF-8 prefix in the string. |
+| sink: fix kafka max message size inaccurate issue            | ❌                 | ❌                                 | ❌                    | Change the default value of Kafka Sink max-message-bytes from 512 MB to 1 MB to prevent TiCDC from sending too large messages to Kafka clusters |
+| cdc/sink: adjust kafka initialization logic                  | ❌                 | ❌                                 | ❌                    | Change the default value of Kafka Sink partition-num to 3 so that TiCDC distributes messages across Kafka partitions more evenly |
+| cmd: hide --sort-dir in changefeed command. (deprecated warning exists) | ✅                 | ❌                                 | ❌                    | Deprecate `--sort-dir` in the `cdc cli changefeed` command. Instead, users can set `--sort-dir` in the `cdc server` command. |
+
+## Bug fix
+
+A bug fix note means that your PR fixes an existing bug or issue. This type of notes start with "Fix" followed by "the issue/bug".
+
+Write your note clearly and adequately so that your target readers can get the main point of your bug fix. The bug or issue must be directly perceivable to the users, and you can refer to the associated GitHub issues.
```

Comment:
Should we also recommend to highlight the bug trigger condition? It may be a good practice.

For example:

✓: Fix the issue that the computed result can be wrong when feature foo is enabled
✗: Fix the issue that the computed result can be wrong

Reply:
Added